storm-core/src/jvm/backtype/storm/scheduler/Topologies.java

```diff
@@ -21,10 +21,17 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import backtype.storm.scheduler.resource.RAS_Component;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 public class Topologies {
     Map<String, TopologyDetails> topologies;
     Map<String, String> nameToId;
-    
+    Map<String, Map<String, RAS_Component>> _allRAS_Components;
+
+    private static final Logger LOG = LoggerFactory.getLogger(Topologies.class);
```

Comment:
This does not look like it is being used.  Not a big deal, but unless we use it we should probably remove it.